src/EditorFeatures/Core.Cocoa/NavigationCommandHandlers/FindBaseSymbolsCommandHandler.cs

```diff
@@ -59,38 +57,37 @@ protected override bool TryExecuteCommand(int caretPosition, Document document,
 
         private async Task StreamingFindBaseSymbolsAsync(
             Document document, int caretPosition,
-            IStreamingFindUsagesPresenter presenter,
-            CancellationToken cancellationToken)
+            IStreamingFindUsagesPresenter presenter)
         {
             try
             {
                 using var token = _asyncListener.BeginAsyncOperation(nameof(StreamingFindBaseSymbolsAsync));
 
-                var context = presenter.StartSearch(EditorFeaturesResources.Navigating, supportsReferences: true, cancellationToken);
+                var (context, cancellationToken) = presenter.StartSearch(EditorFeaturesResources.Navigating, supportsReferences: true);
 
                 using (Logger.LogBlock(
                     FunctionId.CommandHandler_FindAllReference,
                     KeyValueLogMessage.Create(LogType.UserAction, m => m["type"] = "streaming"),
-                    context.CancellationToken))
+                    cancellationToken))
```

Comment:
taht's fine.  because these are all fire-and-forget async methods, they all have an appropriate:

```c#
try
{
    await the async work
}
catch (OperationCanceled)
{
}
catch (Exception ex) when (FatalError.ReportAndCatch(ex))
{
}
```

this makes sure no exceptions leak out, and that we properly NFW report the issues that may arise.